modules/disk-uploader/pkg/vmexport/vmexport.go

```diff
@@ -57,8 +60,17 @@ func WaitUntilVirtualMachineExportReady(client kubecli.KubevirtClient, namespace
 			return false, err
 		}
 
-		if vmExport.Status != nil && vmExport.Status.Phase == v1beta1.Ready {
-			return true, nil
+		if vmExport.Status != nil {
+			log.Logger().Info("VirtualMachineExport object status", zap.String("status", string(vmExport.Status.Phase)))
```

Comment:
Is `zap.String` required here? What does it help with?

Reply:
We already using it here [1] to log the namespace and name, so I thought it would be useful here as well.

[1] https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/cmd/disk-uploader/main.go#L38

Reply:
Hm, I see...